src/init.cpp

```diff
@@ -822,11 +822,6 @@ void InitParameterInteraction()
     }
 }
 
-static std::string ResolveErrMsg(const char * const optname, const std::string& strBind)
```

Comment:
Shoud `#include <util/error.h>` then?

Reply:
Will leave as is for now to not invalidate review